details.browser.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Just a heads up that we'll have to rework this slightly when we merge in @DylanPiercey's changes for ARC. But we can leave as is for now while we test and review.